### PR TITLE
Fixes to BSD-3-Clause license

### PR DESCRIPTION
**Pull request by rrjbca**
## Description
Fixes some inaccuracies in [LICENSE.rst](https://github.com/skypyproject/skypy/blob/master/LICENSE.rst) to bring it in line with [BSD-3-Clause](https://opensource.org/licenses/BSD-3-Clause)
- Clause 3 referred to "the Astropy Team"; now correctly refers to "the copyright holder"
- Clauses are now numbered
- Fixed whitespace and line widths

## Checklist
- [x] Follow the [Contributor Guidelines](https://github.com/skypyproject/skypy/blob/master/CONTRIBUTING.md)
- [x] Assign someone from the infrastructure team to review this pull request

## Questions
- Do we need to update the year to "2020" or "2019-2020"?
- [BSD-3-Clause](https://opensource.org/licenses/BSD-3-Clause) does not include "(c)" or "All rights reserved." do we need to remove them?

### CODE REVIEW

**Comment by rrjbca** [comment]
From [How to use GNU licenses for your own software](http://www.gnu.org/licenses/gpl-howto.html):
"The copyright notice should include the year in which you finished preparing the release"]

File: LICENSE.rst [hunk 1]
```diff
@@ -1,25 +1,26 @@
-Copyright (c) 2019, SkyPy Team
-All rights reserved.
+Copyright 2020 SkyPy Team
```
Comment (rrjbca) [inline LICENSE.rst:1]: I don't think the "SkyPy Team" is formally defined anywhere. I would prefer if we used "The SkyPy Project" (i.e. our [GitHub Organization](https://github.com/skypyproject)) or "SkyPy Developers" (i.e. the [repository contributors](https://github.com/skypyproject/skypy/graphs/contributors))

Comment (ntessore) [inline LICENSE.rst:1]:
```suggestion
Copyright 2020 The SkyPy Project
```

Comment (ntessore) [reply LICENSE.rst:1]:
```suggestion
Copyright 2020 The SkyPy Project
```

Comment (rrjbca) [reply LICENSE.rst:1]: This would be my preference, but I think we should get some consensus from other members before making the change

**Pull request merged**